data-updates.lua

```diff
@@ -76,33 +76,33 @@ if mods["Krastorio2"] then
   data.raw["cargo-wagon"]["space-cargo-wagon"].equipment_grid = "kr-wagons-grid"
   data.raw["fluid-wagon"]["space-fluid-wagon"].equipment_grid = "kr-wagons-grid"
 
-  data.raw["recipe"]["recipe-space-locomotive"].ingredients =
+  data.raw["recipe"]["space-locomotive"].ingredients =
     {{"locomotive", 1}, {"processing-unit", 20}, {"electronic-components", 20}, {"steel-gear-wheel", 20},
      {"rare-metals", 80}}
-  table.insert(data.raw["recipe"]["recipe-space-cargo-wagon"].ingredients, {"steel-gear-wheel", 20})
-  table.insert(data.raw["recipe"]["recipe-space-fluid-wagon"].ingredients, {"steel-gear-wheel", 20})
+  table.insert(data.raw["recipe"]["space-cargo-wagon"].ingredients, {"steel-gear-wheel", 20})
+  table.insert(data.raw["recipe"]["space-fluid-wagon"].ingredients, {"steel-gear-wheel", 20})
```

Comment:
I wonder why it sometimes says `{type="item", name="somethingsomething", amount="10"}` and sometimes it's just `{"somethingsomething", 10}` is the former expression the same but just more verbose?

Reply:
That was the case in 1.1. In 2.0, the latter is invalid. I guess I forgot to change these bits (and they don’t run unless K2 is enabled). They’ll need changing too or else K2 compatibility will be broken.